specification/batch/Azure.Batch/routes.tsp

```diff
@@ -181,12 +181,12 @@ interface Pools {
     BatchPoolHeaders & {
       @doc("An OData $select clause.")
       @query
-      @clientName("select")
+      @clientName("select", "!autorest")
```

Comment:
scope `"!autorest"` doesn't work for Python.

Reply:
@msyyc, can you clarify what you mean? The reasoning for this here is that autorest didn't use to respect this value so we just want to exclude autorest to prevent breaking change to the swagger. If we want to remove `!autorest` and update teh swagger we can do it but I would say as a follow up PR.

Reply:
TCGC may not support negative for scope yet and we will follow the issue.

Reply:
I see, the alternative is to replace with the list of languages for now